### ISSUE TITLE
PlayActor will not appear on Second-Screen app, making set-up impossible 

### ISSUE THREAD

**Issue by pollarliam** [kopiro/homebridge-playstation#73]
**Describe The Bug:**
 PlayActor does not appear in PS4 Second Screen app. 

**To Reproduce:**
 (I have ps4-waker set up, already, maybe this could be causing the issue). To reproduce I simply run homebridge-playstation-login, it discovers my PS4, and PlayActor, when trying to auth to PS4 I'm prompted "Registering with device via Second Screen. Open the PS4 Second Screen app and attempt to connect to the device named:
  PlayActor" however, it does not appear inside of the Second Screen app.

**Expected behavior:**
 PlayActor should appear in the Second Screen app 

**Logs:**

```
 No info related to this plug-in inside of HB logs!
```

**Plugin Config:**

```json
Default
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 1.6.0 
* **Node.js Version**:  v16.16.0 
* **NPM Version**:  8.11.0 
* **Operating System**:  macOS 13

**Comment by pollarliam** [comment]
> I have the same bug. Previously I had already installed PS4 waker with another plugin. I couldn't find a solution :/

Do you by any chance have tried pairing your PS4 using an iOS device? 

Apparently PlayActor won't show in iOS devices (the dev was unaware of this until now)

I'm not sure if PlayActor doesn't appear at all on iOS devices or if it just won't appear in iOS devices if PS4-waker was already installed, I'll test it soon to get more info for the developer 

In the meantime, try using an android device to pair with ps4-walker

**Comment by JorgeHerreraOli** [comment]
ohhh I understand! I don't have an andriod cell phone to test :/

**Comment by pollarliam** [comment]
Shame, maybe try using an android emulator with wi-fi enabled (not sure this will work but it's worth a try!)

**Comment by iandaynes** [comment]
I’m having the same problem. I’ve tried an android phone and PlayActor still doesn’t show up. Could this be down to port forwarding on our routers? Second screen requires some ports to be forwarded to the PS4. Presumably for PlayActor to show up in second screen those same ports will need to be forwarded to wherever homebridge is running?

**Comment by DSimeone1989** [comment]
Any update on this?

**Comment by sidhuamansingh** [comment]
any update ? having similar issues

**Comment by and43a** [comment]
Same problem..

**Comment by denisgabriel5** [comment]
Same problem

**Comment by denisgabriel5** [comment]
Never mind, I found the solution. 

I tried all of these actions at once, so I don't know exactly which one did the trick. But what I've done was:
- Uninstall all the PS4 Waker related plugins from Homebridge UI;
- Remove all the devices from PlayStation (Settings->Mobile App Connection Settings)
- Uninstall ps4-waker from homebridge bash ('npm uninstall ps4-waker -g')
- Re-run from homebridge bash this command: 'homebridge-playstation-login'. After that it instructed me to go Mobile App Connection Settings and provide the PIN code.
